### PR TITLE
Add test: haveResourceServerScopeInAccessTokenWhenThereAreMultiResourceServerScopesInAuthCode

### PR DESCRIPTION
**Pull request by chenrujun**
- Add test: haveResourceServerScopeInAccessTokenWhenThereAreMultiResourceServerScopesInAuthCode
 - Format code.

### CODE REVIEW

File: sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapp/AADWebAppConfigurationTest.java [hunk 1]
```diff
@@ -206,12 +206,12 @@ public void clientRequiresOnDemandPermissions() {
 
     @Test
     public void groupConfiguration() {
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot/src/test/java/com/azure/spring/aad/webapp/AADWebAppConfigurationTest.java:208]: No logic change, just format code.

**Comment by chenrujun** [comment]
Hi, @yiliuTo , please help to review this PR. 🙏

**Comment by chenrujun** [comment]
Hi, @yiliuTo , please help to review this PR. 🙏

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AADWebAppConfiguration.java [hunk 2]
```diff
@@ -86,7 +86,7 @@ private AzureClientRegistration createDefaultClient() {
         return new AzureClientRegistration(client, accessTokenScopes);
     }
 
-    private int resourceServerCount(Set<String> scopes) {
+    public static int resourceServerCount(Set<String> scopes) {
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AADWebAppConfiguration.java:89]: I think only the number of http is not accurate enough, we should count the number of unique urls.

**Comment by chenrujun** [comment]
Hi, @yiliuTo , I added another commit, please review again. 🙏

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**